website/pages/docs/migration.mdx

```diff
@@ -522,6 +522,10 @@ prop instead.
 - Removed `preserveScrollBarGap` prop. We preserve scroll bar gap by default to
   prevent any layout shift.
 
+- `ModalCloseButton` now reads the `onClose` action from the `Modal` context - 
+  you no longer need to pass the `onClick` to it. Conversely, the `onClose` prop
```

Comment:
I don't think we ever required the user to pass `onClose` to the `ModalCloseButton` even in v0.x.

https://v0.chakra-ui.com/modal

I think we can omit this part. Let me know your thoughts.

Reply:
In v0.x, it worked fine if `onClose` is specified in `<ModalCloseButton />` but not in `<Modal />`. But in v1.x+, it breaks. This one threw me for a loop until I dug into the source code! I guess this is a relic from v0.7.

```suggestion
- `ModalCloseButton` now reads the `onClose` action from the `Modal` context. 
   If you had passed your close callback to the `ModalCloseButton.onClick`, you
   need to migrate that callback to be passed into the `Modal.onClose` prop.
```

Would that read better?

Reply:
Yes, thank you